.github/workflows/docker.yml

```diff
@@ -6,6 +6,14 @@ on:
       - main
       - development
   workflow_dispatch:
+    inputs:
+      buildtarget:
+        description: 'Build Target'
+        required: false
+        type: choice
+        options:
+        - development
+        -
```

Comment:
Only note is whether this extra empty dash needs to be here.

Reply:
Yes, it does :-)